tensorflow_datasets/scripts/generate_visualization.py

```diff
@@ -1,18 +1,33 @@
-from matplotlib import figure as fig
-from matplotlib import pyplot as plt
+"""
+Script for generating dataset example figures
+
+Uses show_examples() to generate example figures of a dataset
+and stores it temporary in examples/
+
+Typical usage example:
+
+  ds_name = 'beans'
+  generate_visualization(ds_name)
+
+"""
+
 import tensorflow_datasets as tfds
-from tensorflow_datasets import testing
+import matplotlib.pyplot
 
-figures_dir = ('examples/')
+FIG_DIR = ('examples/')
 
 def generate_visualization(ds_name):
-    try:
-        ds, ds_info = tfds.load(name=ds_name, split='train', with_info=True)
-        fig = tfds.show_examples(ds_info, ds)
-        path_dir = tfds.core.get_tfds_path(figures_dir)
-        fig.savefig(path_dir+ds_name+'.png')
-
-        print('Saved '+ ds_name + '.png' + 'to' + path_dir)
-    except:
-        print('The selected dataset is not supported')
+  """
+  Method used to generate examples figures of a dataset
+  """
+
+  try:
+    ds, ds_info = tfds.load(name=ds_name, split='train', with_info=True)
+    path_dir = tfds.core.get_tfds_path(FIG_DIR)
+    print(path_dir)
+    fig = tfds.show_examples(ds_info, ds, plot_scale=2)
```

Comment:
Using `show_example(plot_scale=2)` the figure size is ~500 kB, is that compact enough or would further optimization be necessary? also following the generated figure
![image](https://user-images.githubusercontent.com/16401940/76794827-2b8cf680-67c8-11ea-89c6-db6c78b083c2.png)

Reply:
> Using show_example(plot_scale=2) the figure size is ~500 kB, is that compact enough or would further optimization be necessary? also following the generated figure

Thank you very much. I think we should try to push the compression a little more. TFDS has 150+ datasets (and growing), most of which are images, and each datasets may have multiple configurations (50+ for [imagenet2012_corrupted](https://www.tensorflow.org/datasets/catalog/imagenet2012_corrupted)). Displaying 100 datasets would took 50MB, which seems to big of an increase.

Could we try to reduce the size of each images to 100Kb or less, and see if this is still readable ?

Reply:
Any update on this ?

Reply:
@Conchylicultor , yes the figures are being optimized. now the image size <100 kB.

Following an example of a image generated https://github.com/tensorflow/datasets/blob/ea30178ea9af8eaaa9717818f4bff0ca077c3c57/docs/catalog/images/beans.jpg

Reply:
Thanks. After discussing internally, we'll store the images on our remote bucket, instead of Github, this way they won't be any size issues. Sorry about this.

Reply:
I'll apply the modifications internally when merging this.